app/code/Magento/Sales/view/frontend/templates/order/creditmemo/items/renderer/default.phtml

```diff
@@ -17,7 +17,7 @@
                 <?php if (!$block->getPrintStatus()) : ?>
                     <?php $_formatedOptionValue = $block->getFormatedOptionValue($_option) ?>
                     <dd<?= (isset($_formatedOptionValue['full_view']) ? ' class="tooltip wrapper"' : '') ?>>
-                        <?= $block->escapeHtml($_formatedOptionValue['value']) ?>
+                        <?= $block->escapeHtml($_formatedOptionValue['value'], ['a', 'img']) ?>
```

Comment:
`img` tag is not allowed in the html Escaper class:
https://github.com/magento/magento2/blob/2.4-develop/lib/internal/Magento/Framework/Escaper.php#L40

This generates an error `The following tag(s) are not allowed: img` when go to the Order page in My Account
The line in the Escaper:
https://github.com/magento/magento2/blob/2.4-develop/lib/internal/Magento/Framework/Escaper.php#L457

Probably removing the `img` from allowed tags passed as the second parameter should be enough.

Reply:
I created pull request, that fixed critical error 
https://github.com/magento/magento2/pull/29863